src/Module/BaseApi.php

```diff
@@ -331,6 +327,28 @@ protected static function getUser($contact_id = null)
 		return api_get_user($contact_id);
 	}
 
+	/**
+	 * Exit with error code
+	 *
+	 * @param int $code
+	 * @param string $description
+	 * @param string $message
+	 * @param string|null $format
+	 * @return void
+	 */
+	public static function error(int $code, string $description, string $message, string $format = null)
+	{
+		$error = [
+			'error'   => $message ?: $description,
+			'code'    => $code . ' ' . $description,
+			'request' => DI::args()->getQueryString()
+		];
+
+		header($_SERVER['SERVER_PROTOCOL'] ?? 'HTTP/1.1' . ' ' . $code . ' ' . $description);
```

Comment:
If `$_SERVER['SERVER_PROTOCOL']` is set, `$code` and `$description` won't be shown.
```suggestion
		header(($_SERVER['SERVER_PROTOCOL'] ?? 'HTTP/1.1') . ' ' . $code . ' ' . $description);
```